packages/edit-post/src/components/start-page-options/index.js

```diff
@@ -83,19 +75,20 @@ function StartPageOptionsModal() {
 			className="edit-post-start-page-options__modal"
 			title={ __( 'Choose a pattern' ) }
 			isFullScreen
-			onRequestClose={ () => setModalState( 'closed' ) }
+			onRequestClose={ onClose }
 		>
 			<div className="edit-post-start-page-options__modal-content">
 				<PatternSelection
 					blockPatterns={ startPatterns }
-					onChoosePattern={ () => setModalState( 'closed' ) }
+					onChoosePattern={ onClose }
 				/>
 			</div>
 		</Modal>
 	);
 }
 
 export default function StartPageOptions() {
+	const [ modalState, setModalState ] = useState( 'initial' );
```

Comment:
It's nice that you've been able to reduce the three-state (`initial`, `open`, `closed`) logic into just two states, "open", "closed". The code could be further simplified to use a single boolean value:
```js
const [ isClosed, setClosed ] = useState( false );
```

Reply:
Makes sense, updated with [8127399](https://github.com/WordPress/gutenberg/pull/55873/commits/8127399017676ff7ade6281b3e4446b20a4b4f9c) 👍